### PR TITLE
Feature/groot 82 reports page to receive information

### PR DESCRIPTION
**Pull request by ahmedlamidi**
Added an endpoint to get banks from the backend using GraphQl.

### CODE REVIEW

File: GraphQL-README.md [hunk 1]
```diff
@@ -135,6 +136,67 @@ query BrandByTagQuery($tag: String!) {
 }
 ```
 
+## individual bank by name
+
+**query**
+
+```graphql
+query BrandByTagQuery($name: String!) {
```
Comment (brugo-dev) [inline GraphQL-README.md:144]: Should be `BrandByNameQuery`

File: GraphQL-README.md [hunk 2]
```diff
@@ -135,6 +136,67 @@ query BrandByTagQuery($tag: String!) {
...
+  brand(name: $name) {
```
Comment (brugo-dev) [inline GraphQL-README.md:145]: should be `brandByName(name: $name)`

File: GraphQL-README.md [hunk 3]
```diff
@@ -135,6 +136,67 @@ query BrandByTagQuery($tag: String!) {
...
+    name
+    website
+    commentary {
+      rating
+      embrace
+      fromTheWebsite
+      ourTake
+      amountFinancedSince2016
+      fossilFreeAlliance
+      top_pick
```
Comment (brugo-dev) [inline GraphQL-README.md:156]: Could you update to `topPick` while you're in here and same thing in the example for brand by tag above

File: GraphQL-README.md [hunk 4]
```diff
@@ -76,7 +76,7 @@ query BrandsQuery(
 }
 ```
 
-## individual bank
+## individual bank by tag
```
Comment (brugo-dev) [inline GraphQL-README.md:79]: Also need to update the line below from
`brand(tag: $tag) {`
to 
`brandByTag(tag: $tag)`

File: GraphQL-README.md [hunk 5]
```diff
@@ -135,6 +136,67 @@ query BrandByTagQuery($tag: String!) {
...
+  }
+}
+```
+
+
+**variables**
+
+```graphql
+{
+  "name": "Ecobank"
```
Comment (brugo-dev) [inline GraphQL-README.md:195]: Better to use a name example that returns result - ex. "Vancity"

**Review by brugo-dev — CHANGES_REQUESTED** [review]
Tested and works. 

Just some errors in the readme examples to fix.

File: brand/schema.py [hunk 6]
```diff
@@ -255,11 +255,16 @@ class Query(graphene.ObjectType):
     commentary = relay.Node.Field(Commentary)
     commentaries = DjangoFilterConnectionField(Commentary)
 
-    brand = graphene.Field(Brand, tag=graphene.Argument(graphene.String, required=True))
+    brand_by_tag = graphene.Field(Brand, tag=graphene.Argument(graphene.String, required=True))
```
Comment (brugo-dev) [inline brand/schema.py:258]: This is a breaking change -- anyone relying on the old endpoint will need to be updated.  Unless you know where this is called and have corresponding updates in those repos I would leave this as is.

File: brand/schema.py [hunk 7]
```diff
@@ -255,11 +255,16 @@ class Query(graphene.ObjectType):
...
 
-    def resolve_brand(root, info, tag):
+    brand_by_name = graphene.Field(Brand, name=graphene.Argument(graphene.String, required=True))
+
+    def resolve_brand_by_tag(root, info, tag):
```
Comment (brugo-dev) [inline brand/schema.py:262]: Same as above - better to leave this endpoint named 'brand' unless you are also updating all the code in other repos that are relying on it.

File: GraphQL-README.md (hunk 5 above)
Comment (ahmedlamidi) [reply GraphQL-README.md:195]: Ecobank is a bank that is available in the database.

File: brand/schema.py (hunk 6 above)
Comment (ahmedlamidi) [reply brand/schema.py:258]: Yeah makes sense, I forgot that this would be called in other places and possibly affect them too.

**Review by brugo-dev — APPROVED** [review]

**Pull request merged**